nsis/README.txt

```diff
@@ -61,25 +71,39 @@ To build the installable .exe:
 	    |	libiconv-2.dll
 	    |	libgcc_s_sjlj-1.dll
 	    |
-	    ` gettext64/
+	    + gettext64/
 		libintl-8.dll
 		libiconv-2.dll
 
-    The default <GETTEXT directory> is "..", however, you can change it by
-    passing /DGETTEXT=... option to the makensis command.
+    The default <GETTEXT directory> is "..", however, you can specify another
+    directory by passing /DGETTEXT=<dir> option to "makensis.exe" program via
+    the command line.
 
 
 Install NSIS if you didn't do that already.
-Also install UPX, if you want a compressed file.
-Download and include the ShellExecAsUser.dll Unicode version which can be
-sourced from: https://nsis.sourceforge.io/ShellExecAsUser_plug-in
+Download Unicode version the ShellExecAsUser plug-in for NSIS from:
+	https://nsis.sourceforge.io/ShellExecAsUser_plug-in
+and put ShellExecAsUser.dll to path\to\NSIS\Plugins\x86-unicode
 
 
 Unpack the images:
 	cd nsis
-	unzip icons.zip
+	unzip icons.zip (on Unix-like)
+	7z.exe x icons.zip or WinRar.exe x icons.zip (on OS Windows)
```

Comment:
There is also Windows version of unzip, and there is also Linux version of 7z (p7zip).

Reply:
I agree. But if in Unix-like systems "unzip" is practically part of the system, then in Windows it is usually only if Gnuwin32 or something similar is installed.
I proceeded from the fact that it is most common in Windows, and 7z is more likely to be installed by the user.
But you're right, it's better to clarify this in the documentation.